### ISSUE TITLE
All my notes and tags have disappeared

### ISSUE THREAD

**Issue by sdclee** [glushchenko/fsnotes#1014]
I had to reboot my iMac Pro as safari was glitching with YouTube videos after the latest Catalina update (maybe nothing to do with the update as it has been working).
After opening FsNotes everything looks to have disappeared.
Attached is a screenshot from below my Library.
Any help would be appreciated.
Thanks in advance.
L
ee
![Screenshot 2020-11-18 at 11 23 33](https://user-images.githubusercontent.com/1264229/99524599-91053880-2990-11eb-9f6a-11c14a7c0357.png)
![Screenshot 2020-11-18 at 11 24 34](https://user-images.githubusercontent.com/1264229/99524658-a8dcbc80-2990-11eb-9692-ed3f2610ce88.png)

**Comment by glushchenko** [comment]
Where did you keep your notes?

**Comment by sdclee** [comment]
That's a great question to which I don't have a definitive answer.
I started out with FsNotes on my iPhone, then iPad, then Mac and they are
all synced via iCloud.
On the Mac, Default storage shows: Users > lee > Library > Containers >
co.fluder.FSNotes > Data > Documents

As you can see from one of the screenshots, that folder only has a shortcut
to iChats and a Trash folder. In Trash there is just a zero byte
goo.markdown.
I also have a folder Saved Application State which has little or nothing of
any substance (datetime stamped recently).
Thanks.
L
ee

**Comment by gingerbeardman** [comment]
Did you use an old version of FSNotes on mac that did tags as Finder?

Or maybe your notes path has changed?

**Comment by glushchenko** [comment]
Default path for iCloud Drive is "~/Library/Mobile Documents/iCloud~co~fluder~fsnotes/Documents"

**Comment by glushchenko** [comment]
@sdclee did you install the update from github or brew?

**Comment by sdclee** [comment]
Many thanks for the *very* rapid responses.
I keep all my apps up to date. The path I described is what is showing in
preferences.
I have a macbook pro downstairs which I'll fetch and check.
All my fsnotes files are indeed in "iCloud Drive / FSNotes" on the iMac.
L
ee

**Comment by sdclee** [comment]
Version 4.8.3 (439) and apologies for not remembering is wasn't from the
App Store.
Almost certainly github but I might be able to prove that one way or the
other. I'll grab the macbook pro.

**Comment by glushchenko** [comment]
@sdclee, i understand. Just change default storage to iCloud Drive / FSNotes if ypu want use GitHub version. But personally i recommend MAS, is more reliable and tested. GitHub versions mostly for dev and tests.

Thanks.

**Comment by sdclee** [comment]
By MAS do you mean by using brew?
Also on my macbook pro where all the notes are visible they are located in
iCloud/FSNotes (so the same as the iMac Pro) but the preferences show:
Users > lee > Library > Mobile Documents >
iCloud~co~fluder~fsnotes~Documents
which is somewhat confusing.
L
ee

**Comment by glushchenko** [comment]
MAS = Mac App Store

**Comment by glushchenko** [comment]
You have 2 options:

1. Install app from Mac App Store this will resolve the issue 
2. Change storage path back to  iCloud Drive / FSNotes.

Thats all.

**Comment by glushchenko** [comment]
Mac App Store and GitHub versions have different storage path. You understand me?

**Comment by sdclee** [comment]
Ok Sorry. I thought you were saying that there are only two ways to
install, either github or brew. I haven't seen MAS before as meaning Mac
Apple Store, so that's where it would have come from.
MAS shows the app as being the latest (i.e. no updates required).
Before I hit this button please advise.
Thanks again.
L
ee

**Comment by sdclee** [comment]
Yes understood and mine is from MAS. I have no idea how the patch got
changed as my notes all sync via icloud across imac, mac, ipad and iphone
(until today).
L
ee

**Comment by glushchenko** [comment]
You said you installed GitHub version, @sdclee! 

> Mac App Store and GitHub versions have different storage path. You understand me? @sdclee

So, if you want, please remove app from /Applications and install from Mac App Store. Thats all.

**Comment by sdclee** [comment]
Apologies. When I said github I thought you meant that FSNotes for the mac
was only available from github or bew and I almost never use brew but I do
use github.
If your question had been "did you install it from github, brew or the app
store" I would definitely have said MAS at that point, so I'm sorry to have
misled you.
Given that it is from the App Store please see my last screenshot and
confirm that it's ok to click the 'Yes' button the resolve my missing notes.
Thanks again.
L
ee

**Comment by glushchenko** [comment]
> 'Yes' button

What?

**Comment by sdclee** [comment]
Ok don't worry - the "Yes" button was at the bottom of my last screenshot.
I pressed it anyway. First time round I got errors/warnings regarding
.DS_store and Trash not being able to be moved because they existed in both
locations but then I changed the path in preferences once more and 'hey
presto!' the notes are back safely :)
I have absolutely no idea how they got to disappear as I have done nothing
other than reboot the iMac. Go figure.
I really appreciate your help and I especially appreciate the rapid
responses.
You are a special person indeed. :)
Many, many thanks.
Take care and stay safe.
L
ee
